### PR TITLE
Create Chandra-Sekhar-Bala.json

### PR DESCRIPTION
**Pull request by Chandra-Sekhar-Bala**
data: Chanra-Sekhar-Bala

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7686"></a>

### CODE REVIEW

File: data/Chandra-Sekhar-Bala.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+		}
+	],
+
+	"links": [{
+			"name": "Connect on Linkedin",
+			"url": "https://www.linkedin.com/in/chandra-sekhar-bala/",
+			"icon": "FaLinkedin"
+		},
+		{
+			"name": "Connet on Twitter",
```
Comment (CBID2) [inline data/Chandra-Sekhar-Bala.json:37]: [Proofreading] 
```suggestion
			"name": "Connect with me on Twitter",
```

File: data/Chandra-Sekhar-Bala.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+			"url": "https://twitter.com/chandrastwt",
+			"icon": "FaTwitter"
+		},
+		{
+			"name": "Github",
```
Comment (CBID2) [inline data/Chandra-Sekhar-Bala.json:42]: [Proofreading]
```suggestion
			"name": "Follow me on GitHub",
```

File: data/Chandra-Sekhar-Bala.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+			"url": "https://github.com/Chandra-Sekhar-Bala",
+			"icon": "FaGithub"
+		},
+		{
+			"name": "Portfolio website",
+			"url": "https://chandra-sekhar-bala.github.io/Portfolio",
+			"icon": "FaLink"
+		},
+		{
+			"name": "Personal Blog",
```
Comment (CBID2) [inline data/Chandra-Sekhar-Bala.json:52]:
```suggestion
			"name": "Follow my personal blog",
```

File: data/Chandra-Sekhar-Bala.json [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+			"url": "https://chandrasekhar.codes",
+			"icon": "SiHashnode"
+		},
+		{
+			"name": "Leetcode - DSA",
+			"url": "https://leetcode.com/Chandra-Sekhar-Bala",
+			"icon": "SiLeetcode"
+		},
+		{
+			"name": "Instagram",
```
Comment (CBID2) [inline data/Chandra-Sekhar-Bala.json:62]: [Proofreading] 
```suggestion
			"name": " Follow me on Instagram",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi, @Chandra-Sekhar-Bala! :) I just gave you some recommendations on how to improve your Linkfree profile. If you need more suggestions, check out the [profile tips guide](https://linkfree.io/docs/profile-tips) on Linkfree's website. Happy Coding! :)

**Comment by Chandra-Sekhar-Bala** [comment]
Hey @CBID2 Thanks for the review . I have made changes as you suggested, please check : )

**Review by CBID2 — APPROVED** [review]
Looks better now @Chandra-Sekhar-Bala! :) Great job and welcome to EddieHub! :)

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by Chandra-Sekhar-Bala** [comment]
Thank for reviewing @CBID2  and @SaraJaoude for adding me toLinkfree. But when I open  [my linkfree](https://linkfree.io/Chandra-Sekhar-Bala) it's showing my old github profile picture. I updated it before making this PR. Please look linto it!

**Pull request merged**